gen/_template/client.tmpl

```diff
@@ -4,6 +4,20 @@
 
 {{- if $.PathsClientEnabled }}
 {{- with $ops := $.Operations }}
+
+// ClientInterface calls operations described by OpenAPI v3 specification.
+type ClientInterface interface {
```

Comment:
nitpick: I feel like
```suggestion
type Invoker interface {
```
sounds better

Reply:
Way better 👍 Changed and regenerated code.